src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -666,13 +674,15 @@ private static void AddIncompleteMembers(ref SyntaxListBuilder<MemberDeclaration
 
         private void ReduceIncompleteMembers(
             ref SyntaxListBuilder<MemberDeclarationSyntax> incompleteMembers,
-            ref SyntaxToken openBraceOrSemicolon,
+            ref SyntaxToken? openBraceOrSemicolon,
             ref NamespaceBodyBuilder body,
-            ref SyntaxListBuilder initialBadNodes)
+            ref SyntaxListBuilder? initialBadNodes)
         {
             for (int i = 0; i < incompleteMembers.Count; i++)
             {
-                this.AddSkippedNamespaceText(ref openBraceOrSemicolon, ref body, ref initialBadNodes, incompleteMembers[i]);
+                var member = incompleteMembers[i];
```

Comment:
yes.  unfortunately the separatelistbuilder's indexer is annotated as returning null back.  So this is necessary.  it might be appropriate to make teh indexer just return T, and have the cases were we use null in a list annotate the builder instead with T?.  But this was expedient :)